<div id="page0" style="width:595.3pt;height:793.7pt">
<p style="top:88.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">classifications on every microarray</span><span style="font-family:YtmmpsAdvTT86d47313+20,serif;font-size:9.8pt">&#x2014;</span><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">this is called tech-</span></p>
<p style="top:100.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">nology bias.</span></p>
<p style="top:112.9pt;left:64.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">Furthermore, not all genes can be detected with equal</span></p>
<p style="top:124.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">reliability, to the extent that some genes are never de-</span></p>
<p style="top:136.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">tected as being</span><span style="font-family:YtmmpsAdvTT86d47313+20,serif;font-size:9.8pt"> &#x2018;</span><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">regulated</span><span style="font-family:YtmmpsAdvTT86d47313+20,serif;font-size:9.8pt">&#x2019;</span><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt"> (the signal never changes).</span></p>
<p style="top:148.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">This is detection bias, which can reflect aspects of the</span></p>
<p style="top:160.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">transcriptomics technology or the sequence of the tran-</span></p>
<p style="top:172.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">script that is being probed.</span></p>
<p style="top:184.9pt;left:64.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">There is also a third and more obvious bias. The tran-</span></p>
<p style="top:196.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">scriptome of a given cell type or tissue is highly special-</span></p>
<p style="top:208.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">ized, to the point that it can be used to determine the</span></p>
<p style="top:220.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">identity of an unknown RNA profile efficiently; this is</span></p>
<p style="top:232.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">referred to as biological bias.</span></p>
<p style="top:244.9pt;left:64.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">We use two published examples to illustrate a very com-</span></p>
<p style="top:256.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">mon problem. The first is a study of transcriptional re-</span></p>
<p style="top:268.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">sponses to disuse in human skeletal muscle [3]. Here, the</span></p>
<p style="top:280.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">reported ontologies reflect the strong biological bias that</span></p>
<p style="top:292.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">muscle has for oxidative metabolism and protein metabol-</span></p>
<p style="top:304.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">ism. Muscle transcriptome profiles were generated from</span></p>
<p style="top:316.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">biopsy material before and after bed rest. Gene functional</span></p>
<p style="top:328.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">classification analysis was applied to a list of differentially</span></p>
<p style="top:340.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">expressed RNAs, produced by applying an uncorrected t-</span></p>
<p style="top:352.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">test to ~40,000 data points. The differentially expressed</span></p>
<p style="top:364.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">gene list was subject to a Fisher</span><span style="font-family:YtmmpsAdvTT86d47313+20,serif;font-size:9.8pt">&#x2019;</span><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">s exact test within IPA,</span></p>
<p style="top:376.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">with the standard IPA database at that time used as the</span></p>
<p style="top:388.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">reference.</span></p>
<p style="top:400.9pt;left:64.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">Such an input gene list is unquestionably flawed. Never-</span></p>
<p style="top:412.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">theless, the subsequent functional classification profile</span></p>
<p style="top:424.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">appeared to be both significant and meaningful, identify-</span></p>
<p style="top:436.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">ing metabolism and protein pathways that are known to</span></p>
<p style="top:448.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">change in disused skeletal muscle [8]. This, in turn,</span></p>
<p style="top:460.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">allowed the authors to claim that the regulated gene list</span></p>
<p style="top:472.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">was an accurate reflection of the experiment and that</span></p>
<p style="top:484.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">more novel features were uncovered by the IPA analysis.</span></p>
<p style="top:496.9pt;left:64.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">A second example is a study of the impact of obesity</span></p>
<p style="top:508.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">on human liver ageing [2]. Here, 657 genes were identi-</span></p>
<p style="top:520.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">fied for which the RNA expression levels correlated with</span></p>
<p style="top:532.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">altered DNA methylation in liver samples from obese in-</span></p>
<p style="top:544.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">dividuals. This list of genes was subject to analysis using</span></p>
<p style="top:556.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">DAVID, and a number of processes were identified, each</span></p>
<p style="top:568.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">of which would be expected if carrying out a GO ana-</span></p>
<p style="top:580.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">lysis on any list of genes expressed in liver tissue versus</span></p>
<p style="top:592.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">the genome. Critically, the GO analysis provided central</span></p>
<p style="top:604.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">support to there being a plausible link between the samples</span><span style="font-family:YtmmpsAdvTT86d47313+20,serif;font-size:9.8pt">&#x2019;</span></p>
<p style="top:616.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:YtmmpsAdvTT86d47313+20,serif;font-size:9.8pt">&#x2018;</span><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">age score</span><span style="font-family:YtmmpsAdvTT86d47313+20,serif;font-size:9.8pt">&#x2019;</span><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt"> and important components of liver biology.</span></p>
<p style="top:628.9pt;left:64.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">In both of these studies, a complex combination of the</span></p>
<p style="top:640.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">three aforementioned biases are at work. Most obvious</span></p>
<p style="top:652.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">is the fact that, functionally, the tissue transcriptome</span></p>
<p style="top:664.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">profiles were already biased towards the factors identi-</span></p>
<p style="top:676.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">fied by the functional enrichment analysis. That is, they</span></p>
<p style="top:688.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">had a greater than average chance of appearing in the</span></p>
<p style="top:700.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">input (</span><span style="font-family:YtmmpsAdvTT86d47313+20,serif;font-size:9.8pt">&#x2018;</span><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">regulated genes</span><span style="font-family:YtmmpsAdvTT86d47313+20,serif;font-size:9.8pt">&#x2019;</span><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">) list. Thus, the significance of</span></p>
<p style="top:712.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">the pathway analysis cannot subsequently validate, for</span></p>
<p style="top:724.9pt;left:56.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">example, the use of 40,000 unpaired t-tests.</span></p>
<p style="top:88.9pt;left:312.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">The issue of utilizing an appropriate background com-</span></p>
<p style="top:100.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">parison list is further compounded in the case of the</span></p>
<p style="top:112.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">muscle analysis [3], as it is impossible to replicate this</span></p>
<p style="top:124.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">analysis exactly because the background reference</span><span style="font-family:YtmmpsAdvTT86d47313+20,serif;font-size:9.8pt"> &#x2018;</span><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">gene</span></p>
<p style="top:136.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">universe</span><span style="font-family:YtmmpsAdvTT86d47313+20,serif;font-size:9.8pt">&#x2019;</span><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt"> used in IPA is frequently updated. This feature</span></p>
<p style="top:148.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">of bespoke databases makes re-evaluation of existing lit-</span></p>
<p style="top:160.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">erature problematic. At a minimum, analyses carried out</span></p>
<p style="top:172.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">with IPA should utilize a defined background gene list</span></p>
<p style="top:184.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">that is included in the publication.</span></p>
<p style="top:196.9pt;left:312.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">Using a free tool like DAVID, it is possible to attempt</span></p>
<p style="top:208.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">to correct for the three sources of sampling bias. This</span></p>
<p style="top:220.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">can be attempted by using a background gene list, repre-</span></p>
<p style="top:232.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">senting the</span><span style="font-family:YtmmpsAdvTT86d47313+20,serif;font-size:9.8pt"> &#x2018;</span><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">universe</span><span style="font-family:YtmmpsAdvTT86d47313+20,serif;font-size:9.8pt">&#x2019;</span><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt"> of possible genes that could be</span></p>
<p style="top:244.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">called as significantly regulated in the experiment. This</span></p>
<p style="top:256.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">list should contain only factors (RNA or protein) that</span></p>
<p style="top:268.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">are both robustly</span><span style="font-family:YtmmpsAdvTT86d47313+20,serif;font-size:9.8pt"> &#x2018;</span><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">probed or sequenced</span><span style="font-family:YtmmpsAdvTT86d47313+20,serif;font-size:9.8pt">&#x2019;</span><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt"> (to avoid tech-</span></p>
<p style="top:280.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">nology and detection bias) and</span><span style="font-family:YtmmpsAdvTT86d47313+20,serif;font-size:9.8pt"> &#x2018;</span><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">called</span><span style="font-family:YtmmpsAdvTT86d47313+20,serif;font-size:9.8pt">&#x2019;</span><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt"> as expressed (to</span></p>
<p style="top:292.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">avoid biological bias) in the experiment. After all, we do</span></p>
<p style="top:304.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">not want to carry out functional enrichment analysis of</span></p>
<p style="top:316.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">a specific tissue simply to be informed that we are study-</span></p>
<p style="top:328.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">ing that tissue!</span></p>
<p style="top:340.9pt;left:312.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">Adjustments for such biases are currently imperfect.</span></p>
<p style="top:352.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">For example, a technology that could detect all tran-</span></p>
<p style="top:364.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">scripts with equal probability (regardless of differences</span></p>
<p style="top:376.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">in abundance) is needed to make such a background</span></p>
<p style="top:388.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">gene list. This could then be used to decide which func-</span></p>
<p style="top:400.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">tional categories (or pathways) are enriched over and</span></p>
<p style="top:412.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">above the bias that is already present in the tissue being</span></p>
<p style="top:424.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">studied. Currently, no technology provides a solution.</span></p>
<p style="top:436.9pt;left:312.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">We raised this issue with the providers of</span><span style="font-family:YtmmpsAdvTT86d47313+20,serif;font-size:9.8pt"> &#x2018;</span><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">Ingenuity</span><span style="font-family:YtmmpsAdvTT86d47313+20,serif;font-size:9.8pt">&#x2019;</span></p>
<p style="top:448.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">(a commercial web-based application for functional ana-</span></p>
<p style="top:460.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">lysis of</span><span style="font-family:YtmmpsAdvTT86d47313+20,serif;font-size:9.8pt"> &#x2018;</span><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">-omics</span><span style="font-family:YtmmpsAdvTT86d47313+20,serif;font-size:9.8pt">&#x2019;</span><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt"> data) in 2011, and since then, they have</span></p>
<p style="top:472.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">allowed users to submit an experiment-specific back-</span></p>
<p style="top:484.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">ground file (it is still important that researchers publish</span></p>
<p style="top:496.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">a record of the final list of genes that were mapped to</span></p>
<p style="top:508.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">the IPA database). Likewise, DAVID allows the user to</span></p>
<p style="top:520.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">submit a custom background file (although there is no</span></p>
<p style="top:532.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">clear guidance about its essential role). However, some-</span></p>
<p style="top:544.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">what surprisingly, the Gene Ontology consortium web-</span></p>
<p style="top:556.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">site analysis tool [9] does not allow for this option,</span></p>
<p style="top:568.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">making any analysis thoroughly unreliable.</span></p>
<p style="top:580.9pt;left:312.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">Thus, the seriousness of the bias issue remains substan-</span></p>
<p style="top:592.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">tially underappreciated, and guidance to reviewers and</span></p>
<p style="top:604.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">editorial boards on the importance of the three sources of</span></p>
<p style="top:616.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">sampling bias is absent. We have noted in our studies that</span></p>
<p style="top:628.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">correction tends to reduce or completely remove the sig-</span></p>
<p style="top:640.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">nificance of functional terms, and this can be an unwel-</span></p>
<p style="top:652.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">come observation after time and money has been spent</span></p>
<p style="top:664.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">on collecting and analyzing the data. Subjectively, func-</span></p>
<p style="top:676.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">tional enrichment analysis bias occurs very frequently in</span></p>
<p style="top:688.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">clinical genomic studies. This is typically because the</span></p>
<p style="top:700.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">sample size is limited, leading to loosening of statistical</span></p>
<p style="top:712.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">thresholds for differential expression, which is</span><span style="font-family:YtmmpsAdvTT86d47313+20,serif;font-size:9.8pt"> &#x2018;</span><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">compen-</span></p>
<p style="top:724.9pt;left:304.7pt;line-height:9.8pt"><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt">sated</span><span style="font-family:YtmmpsAdvTT86d47313+20,serif;font-size:9.8pt">&#x2019;</span><span style="font-family:GpkmnrAdvTT86d47313,serif;font-size:9.8pt"> [sic] by the use of functional enrichment analysis.</span></p>
<p style="top:34.0pt;left:56.7pt;line-height:8.0pt"><span style="font-family:RllhnpAdvTTe45e47d2,serif;font-size:8.0pt">Timmons</span><span style="font-family:BlcybwAdvTT7329fd89.I,serif;font-size:8.0pt"> et al. Genome Biology</span><span style="font-family:MyriadPro,serif;font-size:8.0pt">  (2015) 16:186 </span></p>
<p style="top:34.0pt;left:500.1pt;line-height:8.0pt"><span style="font-family:RllhnpAdvTTe45e47d2,serif;font-size:8.0pt">Page 2 of 3</span></p>
</div>
